llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -741,6 +747,36 @@ bool isTargetCompatibleWithModule(const std::string &Target,
   return true;
 }
 
+std::optional<module_split::ModuleDesc>
+makeDummy(module_split::ModuleDesc &MD) {
+  bool hasVirtualFunctions = false;
```

Comment:
```suggestion
  bool HasVirtualFunctions = false;
```

[LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)